edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -77,19 +94,58 @@ def authenticate(self, request):
             # CSRF passed validation with authenticated user
             return user_and_auth
 
-        except jwt.InvalidTokenError as token_error:
-            # Note: I think this case is not used, but will monitor the custom attribute to verify.
```

Comment:
**Note to Reviewer:** As noted in the comment, this case was not appearing in New Relic. Instead, the below case is sometimes reporting: `Exception:AuthenticationFailed('Invalid token.')`. This case was removed to simplify the code, including the "original" code.